exchange/bidder.go

```diff
@@ -308,17 +308,19 @@ func getAssetByID(id int64, assets []nativeRequests.Asset) (nativeRequests.Asset
 func makeExt(httpInfo *httpCallInfo) *openrtb_ext.ExtHttpCall {
 	if httpInfo.err == nil {
 		return &openrtb_ext.ExtHttpCall{
-			Uri:          httpInfo.request.Uri,
-			RequestBody:  string(httpInfo.request.Body),
-			ResponseBody: string(httpInfo.response.Body),
-			Status:       httpInfo.response.StatusCode,
+			Uri:            httpInfo.request.Uri,
+			RequestBody:    string(httpInfo.request.Body),
+			ResponseBody:   string(httpInfo.response.Body),
+			Status:         httpInfo.response.StatusCode,
+			RequestHeaders: httpInfo.request.Headers,
```

Comment:
Log Request Headers when this is no error

Reply:
Nitpick: Would you be able to move the `RequestHeaders` definition directly below the `RequestBody`?

Reply:
Moved `RequestHeaders ` below `RequestBody`

Reply:
Moved `RequestHeaders ` below `RequestBody`